basin3d/core/schema/enum.py

```diff
@@ -55,6 +55,7 @@ class TimeFrequencyEnum(str, BaseEnum):
     HOUR = "HOUR"
     MINUTE = "MINUTE"
     SECOND = "SECOND"
+    NONE = "NONE"
```

Comment:
The TimeMetadataMixin in the models.py module which defines aggregation_duration, should be updated to include NONE. Look in lines 1263-79.